extensions/mock-credentials-verifier/src/main/java/org/eclipse/dataspaceconnector/mvd/MockCredentialsVerifier.java

```diff
@@ -66,9 +84,29 @@ public Result<Map<String, Object>> verifyCredentials(String hubBaseUrl, PublicKe
                             s -> (Object) decode(s[1], UTF_8)
                     ));
             monitor.debug("Completing (mock) credential verification. Claims: " + claims);
-            return Result.success(claims);
+            return Result.success(toMappedVerifiableCredentials(claims));
         } catch (MalformedURLException e) {
             throw new EdcException(e);
         }
     }
+
+    private Map<String, Object> toMappedVerifiableCredentials(Map<String, Object> regionClaims) {
+        var vcId = UUID.randomUUID().toString();
+        return Map.of(vcId,
+                Map.of(VERIFIABLE_CREDENTIALS_KEY, Map.of(CREDENTIAL_SUBJECT_KEY, regionClaims,
+                                VERIFIABLE_CREDENTIAL_ID_KEY, vcId),
+                        // issuer will be ignored when applying policies for now.
+                        ISSUER_KEY, String.join("did:web:", UUID.randomUUID().toString())));
```

Comment:
nit: just for readability. Up to you to decide. Thanks
```suggestion
        return Map.of(vcId,
                Map.of(VERIFIABLE_CREDENTIALS_KEY,
                        Map.of(CREDENTIAL_SUBJECT_KEY, regionClaims,
                                VERIFIABLE_CREDENTIAL_ID_KEY, vcId
                        ),
                        // issuer will be ignored when applying policies for now.
                        ISSUER_KEY, String.join("did:web:", UUID.randomUUID().toString())
                )
        );
```